packages/@azure/servicebus/data-plane/README.md

```diff
@@ -1,22 +1,30 @@
-# @azure/service-bus
+# Azure Service Bus client library for Typescript and Javascript
 
-This library provides a convenient way to interact with [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
+Microsoft Azure Service Bus is a highly-reliable cloud messaging service between applications and services.
+
+Use the client library for Azure Service Bus to 
+- Send messages to a Queue or Topic
+- Receive messages from a Queue or Subscription
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/packages/%40azure/servicebus/data-plane) | [Package (npm)](https://www.npmjs.com/package/@azure/service-bus) | [Product documentation](https://azure.microsoft.com/en-us/services/service-bus/)
 
 ## Status
 
-This library is currently in preview and may change prior to release.
+This library is currently in preview and the APIs may change prior to release.
 
-## Pre-requisite
+## Getting Started
 
-- **Node.js version: 6.x or higher.**
+### Install the package
 
-## Installation
+Install the Azure Service Bus client library for Typescript/Javascript using npm
```

Comment:
Agree it's awkward to say this full incantation. Can we just call it "the library"?

Reply:
yes, can you update the general guidelines?

Reply:
Updated to 

> Install the Azure Service Bus client library using npm